specification/columns/chargeclass.md

```diff
@@ -32,6 +32,7 @@ Allowed values:
 | :--------- | :------------------------------------|
 | Regular    | Standard charges for services used or purchased. |

```

Comment:
We should change "Regular" if we're differentiating it from charges that are based on a percentage of other charges. Would "Direct" make sense?

Reply:
I like the term Direct

Reply:
Another proposition (brainstorming) for Regular: Original.

Indeed, a charge maybe be either the original charge or a correction charge.

Reply:
Thinking further, ChargeOccurence might be more explicit than ChargeClass.

Indeed, a charge occurence may be either the original occurrence or a correction occurence.